tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html

```diff
@@ -45,6 +45,11 @@
 -->
 
 <dom-module id="tf-profile-dashboard">
+  <style>
+.profile-btn {
+  width: 160px;
```

Comment:
Why is this explicit width needed?

Reply:
I was thinking to make the button in the side bar wider. It turns out having no effect, so I removed this.